backend/src/main/java/com/kinsella/kinsellaBackend/controllers/ProductsController.java

```diff
@@ -29,17 +38,66 @@ public Collection<Products> getProductByID(@PathVariable Long productID) {
         return productsRepository.findByProductID(productID);
     }
 
+    /*
+    @GetMapping({"/{ticketId}/comment/{commentNumber}", ""})
+    public ResponseEntity getTicketComment(@PathVariable(name = "ticketId") Long ticketId, @PathVariable(name = "commentNumber") Long commentNum){
+    */
+
+    @PostMapping(path = "/admin/add/{name}/{price}/{stock}")
+    public String newProduct (@RequestParam("image") MultipartFile file,
+                              @PathVariable(name = "name") String name,
+                              @PathVariable(name = "price") Double price,
+                              @PathVariable(name = "stock") Integer stock) throws IOException{
```

Comment:
Just to add on the comment above, you can replace these parameters with a single 'Product' object that is passed in from the client and just save that straight to the database without needing lines 53 or 64-67.

Reply:
Will check out in future